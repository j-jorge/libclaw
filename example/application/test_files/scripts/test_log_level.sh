#!/bin/sh

./ex-application -b --string="my string" one two three --log-level=0
./ex-application -b --string="my string" one two three --log-level=1
./ex-application -b --string="my string" one two three --log-level=15
