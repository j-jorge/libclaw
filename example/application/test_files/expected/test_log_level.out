Testing error_level:
error: error_level:program_name == ./ex-application
Testing warning_level:
Testing verbose_level:
'-b' argument is present.
'--string' argument value is 'my string'
Testing error_level:
error: error_level:program_name == ./ex-application
Testing warning_level:
warning: warning_level:argument 0 == one
warning: warning_level:argument 1 == two
warning: warning_level:argument 2 == three
Testing verbose_level:
'-b' argument is present.
'--string' argument value is 'my string'
Testing error_level:
error: error_level:program_name == ./ex-application
Testing warning_level:
warning: warning_level:argument 0 == one
warning: warning_level:argument 1 == two
warning: warning_level:argument 2 == three
Testing verbose_level:
verbose_level:m_app is running
'-b' argument is present.
'--string' argument value is 'my string'
